docs/creating-studies.md

```diff
@@ -127,16 +147,27 @@ styling.
   to find other problems if you lightly adhere to this from the start.
   - Agggregate count tables should have the first word after the study prefix be
   `count`, and otherwise the word `count` should not be used.
+
+**Metadata tables**
   - Creating a table called `my_study__meta_date` with two columns, `min date` and
   `max date`, and populating it with the start and end date of your study, will
   allow other Cumulus tools to detect study date ranges, and otherwise bakes the
   study date range into your SQL for future reference.
+  - Creating a `my_study__meta_version` with one column, `data_package_version`, and
+  giving it an integer value as shown in this snippet:
+  ```sql
+  CREATE TABLE my_study__meta_version AS
+  SELECT 1 AS data_package_version;
+  ```
+  allows you to signal versions for use in segregating data upstream, like in the
+  Cumulus aggregator - just increment it when you want third parties to start running
+  a new data model. If this is not set, the version will implicitly be set to zero.
```

Comment:
"when you want third parties to start running a new data model" -> a little confusing to my eyeballs. Maybe "when third parties using your study will need to re-run it" or "when your data model changes" or something better you come up with

Reply:
tied to counts output.